src/realm/sort_descriptor.hpp

```diff
@@ -66,9 +66,17 @@ class BaseDescriptor {
         {
             return index_in_view < other.index_in_view;
         }
+        Mixed get_value() const
```

Comment:
Open to change this, I just need a way to fetch the value in the same way in the templated comparison function.